python/mead/downloader.py

```diff
@@ -0,0 +1,300 @@
+import sys
```

Comment:
Most python people put all the imports up here rather than in the functions that use them